### ISSUE TITLE
Major issue! After building a static application, the chart is not updating!

### ISSUE THREAD

**Issue by jokerstudios** [apexcharts/vue-apexcharts#163]
When I run the application locally everything works fine.
When I do ```npm run build``` and serve it locally with ```serve -l dist``` everything works fine.
As soon as I build the application with ```npm run build``` and serve it as static content on my server I get the following behavior.

I select data from a dropdown and the plot works as it should be.
I select different data from the dropdown and i keeps showing me the plot from the beginning.
I'm logging the data which are fed into the plot and they are the new updated data.

So basically, the chart only plots whatever gets selected after a site refresh. Every selection after the first one is ignored by the chart.

Can you please look into this issue?

Thanks a lot
